### PR TITLE
feat: Create the "Happy Path" test for the "Openshift Connector" plugin

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
Create the "Happy Path" test for the "Openshift Connector" plugin

##Scenario:
 - Import sample project
 - Go to the "Openshift Connector" tab
 - Expand tree
 - Create deployment using imported sample
 - Check that component is successfully deployed

https://user-images.githubusercontent.com/22724619/151194621-ccea37e5-f64e-4f9a-ad1d-fd8cabbd29cc.mp4

### CODE REVIEW

**Comment by jeffmaury** [comment]
Not sure why you have a bin folder if you followed instructions to load the project into IJ but those files should be removed from the PR

File: build.gradle [hunk 1]
```diff
@@ -9,8 +9,12 @@ plugins {
 }
 
 repositories {
-    maven { url 'https://repository.jboss.org' }
+    mavenLocal()
     mavenCentral()
+    maven { url 'https://repository.jboss.org' }
+    maven {
+        url 'https://packages.jetbrains.team/maven/p/ij/intellij-dependencies'
```
Comment (jeffmaury) [inline build.gradle:16]: Why is this required ?

File: build.gradle [hunk 2]
```diff
@@ -53,7 +57,7 @@ sourceSets {
 }
 
 task integrationTest(type: Test) {
-    systemProperties['com.redhat.devtools.intellij.telemetry.mode'] = 'debug'
```
Comment (jeffmaury) [inline build.gradle:56]: Why is this removed ?

File: build.gradle [hunk 3]
```diff
@@ -96,6 +101,7 @@ runIde {
 }
 
 runIdeForUiTests {
+    systemProperty "robot-server.port", System.getProperty("robot-server.port")
```
Comment (jeffmaury) [inline build.gradle:104]: Can't we use a default value in case the system property is not set

File: src/it/java/org/jboss/tools/intellij/openshift/BaseTest.java [hunk 4]
```diff
@@ -11,29 +11,69 @@
 package org.jboss.tools.intellij.openshift;
 
 import com.intellij.openapi.project.Project;
-import com.intellij.testFramework.fixtures.CodeInsightTestFixture;
```
Comment (jeffmaury) [inline src/it/java/org/jboss/tools/intellij/openshift/BaseTest.java:14]: You removed all CLI test that we use to make sure new version of Odo don't break. Please restore them and use another class as parent for your new UI tests

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (odockal) [reply build.gradle:16]: Part of setup guide: https://github.com/redhat-developer/intellij-common-ui-test-library#step-1-adding-repositories

**Comment by odockal** [comment]
@jeffmaury Cannot push to this PR, do not have rights, I have opened https://github.com/redhat-developer/intellij-openshift-connector/pull/387 in favor of this PR.
